.eslintrc.json

```diff
@@ -109,6 +110,7 @@
 		"no-magic-numbers": [
 			"warn",
 			{
+			        "ignore": [0, 1, 2],
```

Comment:
please make sure the indent here is consistent with below (seems it's a tab too much?)